src/sparsezoo/models/zoo.py

```diff
@@ -186,6 +190,68 @@ def load_model_from_stub(
             override_parent_path=override_parent_path,
         )
 
+    @staticmethod
+    def load_model_from_recipe(
+        recipe: OptimizationRecipe,
+        override_folder_name: Union[str, None] = None,
+        override_parent_path: Union[str, None] = None,
+        force_token_refresh: bool = False,
+    ):
+        """
+        Loads the model associated with a recipe
+
+        :param recipe: the Recipe associated with the model
+        :param override_folder_name: Override for the name of the folder to save
+            this file under
+        :param override_parent_path: Path to override the default save path
+            for where to save the parent folder for this file under
+        :param force_token_refresh: True to refresh the auth token, False otherwise
+        :return: The requested Model instance
+        """
+        metadata = recipe.model_metadata
+        return Zoo.load_model(
+            domain=metadata.domain,
+            sub_domain=metadata.sub_domain,
+            architecture=metadata.architecture,
+            sub_architecture=metadata.sub_architecture,
+            framework=metadata.framework,
+            repo=metadata.repo,
+            dataset=metadata.dataset,
+            training_scheme=metadata.training_scheme,
+            optim_name=metadata.optim_name,
+            optim_category=metadata.optim_category,
+            optim_target=metadata.optim_target,
+            release_version=metadata.release_version,
+            override_folder_name=override_folder_name,
+            override_parent_path=override_parent_path,
+            force_token_refresh=force_token_refresh,
+        )
```

Comment:
`model_metadata` is a subclass of ModelArgs so we should be able to feed this directly into `load_model_from_stub`